### PR TITLE
Feature/mobile build

### PR DESCRIPTION
**Pull request by Alex-Smith-Arcweb**
Update survey app 'app name'

FIx capacitor keyboard calls being fired on web

### CODE REVIEW

File: examples/survey-app/src/app/pages/login/login.page.ts [hunk 1]
```diff
@@ -55,11 +55,17 @@ export class LoginPage implements OnInit {
...
+        await Keyboard.hide();
+      }
       this.router.navigate(['/surveys']);
     } catch (error) {
       this.errorMessage = 'Invalid login credentials';
-      await Keyboard.hide();
+
+      if (Capacitor.getPlatform() !== 'web') {
+        await Keyboard.hide();
+      }
```
Comment (mccolin) [inline examples/survey-app/src/app/pages/login/login.page.ts:68]: @Alex-Smith-Arcweb is this solving a bug we hadn't reported?

**Review by mccolin — APPROVED** [review]
This looks like this addresses the application name issue 👍

File: examples/survey-app/src/app/pages/login/login.page.ts (hunk 1 above)
Comment (Alex-Smith-Arcweb) [reply examples/survey-app/src/app/pages/login/login.page.ts:68]: Correct, sorry I meant to put this in the description.  If the keyboard hide is called on web it errors out

**Comment by mccolin** [comment]
Looks good. Let's make sure our next build keep the same version number we're using currently and ups the build number to 4 (at least according to this doc it'd be the fourth build: https://arcweb.atlassian.net/wiki/spaces/Arcwell/pages/4162289715/Example+Applications)

**Pull request merged**